middleware.md

```diff
@@ -185,6 +186,27 @@ Middleware groups may be assigned to routes and controller actions using the sam
 
 > {tip} Out of the box, the `web` middleware group is automatically applied to your `routes/web.php` file by the `RouteServiceProvider`.
 
+<a name="sorting-middleware"></a>
+### Sorting Middleware
+
+In some cases you'd want to run your middleware in a specific order. You can achieve this by ordering the middleware with the `$middlewarePriority` property of your HTTP kernel. This doesn't includes global middleware which is always run before other middlewares.
```

Comment:
Some typos:

middleware, not middlewares
include, not includes

:)

Reply:
Updated. Thanks!